extension/bearertokenauthextension/README.md

```diff
@@ -13,19 +13,26 @@ The authenticator type has to be set to `bearertokenauth`.
 
 ## Configuration
 
-The following is the only setting and is required:
+One of the following two options is required. If a token **and** a tokenfile are specified, the token is **ignored**:
 
 - `token`: static authorization token that needs to be sent on every gRPC client call as metadata.
   This token is prepended by "Bearer " before being sent as a value of "authorization" key in
   RPC metadata.
-  
-  **Note**: bearertokenauth requires transport layer security enabled on the exporter.
+
+- `tokenfile`: filename of file that contains a authorization token that needs to be sent on every
+  gRPC client call as metadata.
+  This token is prepended by "Bearer " before being sent as a value of "authorization" key in
+  RPC metadata.
+
+
+**Note**: bearertokenauth requires transport layer security enabled on the exporter.
 
 
 ```yaml
 extensions:
   bearertokenauth:
     token: "somerandomtoken"
+    token_filename: "file-containing.token"
```

Comment:
How about just `filename`? Can you check which names we use for the TLS cert files? I have a feeling `filepath` is also used elsewhere.

Reply:
Looks like we have everything ^^. I used `grep -ir "filepath[^.][^\"]" | grep -iv "_test"` to get an overview. There is `filepath`, `filename`, `fileName` and others in use. The `pulsarexporter` uses `TLSTrustCertsFilePath` - `tls_trust_certs_file_path`. 

What do you think of `TokenFilePath` - `token_file_path`?

Reply:
Do we need the "token" prefix? If I read the "filename" property for the "bearertokenauth" component, I assume the file contains the token to use. But perhaps that's just me?